src/commentscan.l

```diff
@@ -706,15 +712,35 @@ STopt  [^\n@\\]*
                                         }
 <Comment>"<"{SUMMARY}">"                { // start of a .NET XML style brief description
                                           if (!yyextra->HTMLDetails) setOutput(yyscanner,OutputBrief);
-                                          addOutput(yyscanner,yytext);
+                                          else
+                                          {
+                                            int id = yyextra->HTMLDetailsSummary[yyextra->HTMLDetails-1];
+                                            if (id == 0)
+                                            {
+                                              addOutput(yyscanner,"\\htmlonly </p> \\endhtmlonly");
```

Comment:
What about other output formats? 

I think this should preferably be handled in HtmlDocVisitor. Do you have an example of the problem, so I can have a look?

Reply:
I didn't get problems with other formats, though I think the entire construction with `\htmlonly` can be omitted when it is possible to move the content of the first `<summary.` style change to the beginning of the handling of the `<details>` in the function `int DocHtmlDetails::parse(DocNodeVariant *thisVariant)` of `docnode.cpp `, but I don't yet know how to do this properly.